app/main/controllers/items.py

```diff
@@ -0,0 +1,125 @@
+import json
+
+from flask import request, abort, jsonify
+from sqlalchemy import String, BOOLEAN
```

Comment:
Change to `Boolean`

Reply:
Fixed, but i had to use bool when checking that the input of is_note/is_highlight are the correct type (will take out is_highlight tomorrow)